### DISCUSSION TITLE
Can't seem to get package:test working

### DISCUSSION THREAD

**Discussion by duncanmcclean** [orchestral/testbench#297] [Q&A]
- Testbench Version: v6.13.0
- Laravel Version: v8
- PHP Version: 7.4.15 (also happens on 8.0.2)
- Database Driver & Version: None

### Description:

I'm investigating whether it's possible for my Statamic addon (which is a Laravel package, at it's core) to use parallel testing. I'm currently seeing my test suite run for 10mins in CI, which isn't ideal.

At the moment, I'm just using plain `./vendor/bin/phpunit` to run my tests. However, it seems that in order to use parallel tests as part of Testbench I need to use the `package:test` command with the `--parallel` parameter.

However, when I try to run that command (with or without that parallel parameter), an exception is thrown.

```
./vendor/bin/testbench package:test
```

```
In TestCommand.php line 129:
                                                                          
Call to undefined method Dotenv\Repository\RepositoryBuilder::create() 
```

If it helps, [this is the repo](https://github.com/doublethreedigital/simple-commerce) of the addon/package I'm working with.

### Steps To Reproduce:

1. Update Testbench to latest version
2. Run `./vendor/bin/testbench package:test`
3. Exception is thrown

**Comment by crynobone** [comment]
Only Testbench 6 has parallel testing support.

**Comment by duncanmcclean** [comment]
I'm using Testbench 6 when I'm testing..

**Comment by crynobone** [comment]
No issue running `package:test` or `package:test --parallel`. See https://github.com/orchestral/testbench-core/actions/runs/622768806

**Comment by besanek** [comment]
@duncanmcclean Try to run `./vendor/bin/testbench package:discover` first. I had same issue.

**Reply by duncanmcclean** [reply]
> ./vendor/bin/testbench package:discover

That doesn't seem to help, unfortunately. 😞  Thanks though!
